ManuProjectFiles/app/src/test/java/com/example/manu/ExampleUnitTest.kt

```diff
@@ -285,5 +285,29 @@ class ExampleUnitTest {
         }
     }
 
+    @Test
+    fun noDuplicatesInBirdDatabase(){
+        var stillValid = true
+        BirdDatabase.compileDatabase()
+
+        val birds = BirdDatabase.getBirdList()
```

Comment:
Consider using birds1 and birds2 because it seems like we didn't anticipate using the second list. For example, Die Hard, Die Hard 2. The sequel wasn't initially planned.